docs/sdks/client-sdks/flutter/bandits.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: Contextual Bandits
+sidebar_position: 5
+---
+
+## Usage with Contextual Multi-Armed Bandits
+
+Eppo supports contextual [multi-armed bandits](/contextual-bandits). You configure bandit flags in the Eppo application, where you can set up:
+
+- Flag keys
+- Status quo variations
+- Bandit variations
+- Targeting rules
+
+When you query the bandit in your code, you provide the available actions to the SDK.
+
+To leverage bandits using the Dart SDK, there are two additional steps over regular feature flags:
+
+1. Implement a logger that handles bandit actions
+2. Query the bandit for an action
```

Comment:
For the precomputed client, there is a third step which is supply the available bandit actions when initializing